packages/fx-core/src/plugins/resource/bot/v2/index.ts

```diff
@@ -67,24 +67,21 @@ export class BotPluginV2 implements ResourcePlugin {
   }
 
   async scaffoldSourceCode(ctx: Context, inputs: Inputs): Promise<Result<Void, FxError>> {
```

Comment:
There is an "Code scanning" warning of Unused variable, import, function or class for the import part, please check with https://github.com/OfficeDev/TeamsFx/security/code-scanning/953